packages/database/test/helpers/util.ts

```diff
@@ -48,14 +48,14 @@ export const DATABASE_URL = USE_EMULATOR
 
 console.log(`USE_EMULATOR: ${USE_EMULATOR}. DATABASE_URL: ${DATABASE_URL}.`);
 
-const qs = {};
-if ('location' in this) {
-  const search = (this.location.search.substr(1) || '').split('&');
-  for (let i = 0; i < search.length; ++i) {
-    const parts = search[i].split('=');
-    qs[parts[0]] = parts[1] || true; // support for foo=
-  }
-}
+// const qs = {};
```

Comment:
Can this block of code be removed? It populates qs with the querystring parameters, but qs is never used.